apps/web/assets/js/components/Filters.jsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react'
 
-const toggleLevels = function(levels, level) {
+const updateLevels = (levels, level) => {
```

Comment:
What a great approach! I thought that it would be a nice idea to get of rid of one of those almost duplicate functions.